src/program/apis/tvdb_api.py

```
@@ -60,361 +113,233 @@ def __init__(self):
+        response = self.session.post("login", json=payload)
+
+        if not response.ok:
+            logger.error(f"Failed to obtain TVDB token: {response.status_code}")
+            return None
+
+        from schemas.tvdb import LoginPost200ResponseData
+
+        data = LoginPost200ResponseData.model_validate(response.json()["data"])
+
+        if not data.token:
+            logger.error(f"Failed to obtain TVDB token: No token in response")
```

Ruff (0.14.6)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)